packages/hash/shared/src/createProseMirrorState.ts

```diff
@@ -42,6 +42,10 @@ export const createProseMirrorState = ({
       );
       return true;
     },
+    // execCommand is flagged as depecrated but it seems that there isn't a viable alternative
+    // to call the undo and redo default browser actions (https://stackoverflow.com/a/70831583)
```

Comment:
Please can we inline comment/note that we expect the below code to be removed with forthcoming collab rework?

Reply:
Done